css/style.css

```diff
@@ -0,0 +1,43 @@
+@media (max-width: 800px){
+    body{
+    background-color: white;
```

Comment:
Use linters formatting. [`Details are here!`](https://github.com/microverseinc/linters-config)

Reply:
- [ ] Please use polite language that does not sound like giving orders. 
`This reads as an order: "Use linters formatting"`
Instead i suggest you use polite language like: 
` "I think it would be better to add linters for formatting"`
Using polite language makes it conversational and the other person feels receptive to your review. Also your job here is to help another person improve in a friendly way.

- [ ] Also kindly explain why its important for the person to use the suggestion you made. This ensures that he/she understands why you requested the changes and helps them improve the overall quality of their code. Also it might be obvious for you but not for someone else. Kindly refer to your lesson: [How to give a code review](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/give_code_review_basics.md)

<img width="1654" alt="Screenshot 2022-10-31 at 12 05 31" src="https://user-images.githubusercontent.com/48546992/198972330-8b3520d8-bd18-4984-b392-4a01c4135f19.png">